TFM samples add 9151 support (#15307)
nrfconnect/sdk-nrf: samples/tfm/provisioning_image/README.rst

Testing
=======

After programming the sample, the following output is displayed in the console:

.. code-block:: console

    Successfully verified PSA lifecycle state ASSEMBLY!
    Successfully switched to PSA lifecycle state PROVISIONING!
    Generating random HUK keys (including MKEK)
    Writing the identity key to KMU
    Success!

If an error occurs, the sample prints a message and raises a kernel panic.

.. note::
   The device cannot transition from the PRoT security lifecycle state **PRoT Provisioning** to the **Device Assembly and Test** state.

>    To reproduce the sample, the device must be flashed with the ``--erase`` option to erase the flash memory before flashing the sample.

Reviewer: ```suggestion
   To reproduce the sample, you must program the device with the ``--erase`` option to erase the flash memory.
```

Use program to indicate the step of putting the sample onto the device in cases where we have 'flash' with two different meanings in the same sentence.
